docs/getting_started/discover_ui.md

```diff
@@ -14,15 +14,20 @@ Depending on your location within the Back Office, it can contain the following
 - Global search field that allows you to [find content](../search/search_for_content.md) by checking all searchable Fields
 - Drop-down that changes the site context
 - Notification icon that informs you about incoming assignments, for example, items assigned for [your review](../content_management/workflow_management/editorial_workflow.md#review-queue)
+- App switcher which allows to quickly switch between applications within the QNTM group ecosystem
```

Comment:
@juskora in practice and in the current state, "app" switcher includes links to QNTM companies official websites (not links to backoffice apps). I would change this sentence a bit e.g.:
_..which includes/presents the links to official websites of companies - world-class ad-tech and martech solutions providers being part of QNTM ecosystem._
or in other form:
__"get to know other world-class mar-tech and ad-tech solutions offered by companies being part of QNTM ecosystem"_